ts/Dashboard/Actions/Bindings.ts

```diff
@@ -111,12 +123,17 @@ class Bindings {
             case 'HTML':
                 component = HTMLComponent.fromJSON(json as HTMLComponent.ClassJSON);
                 break;
-            default:
+            case 'Highcharts': 
                 const componentClass = Component.getComponent(json.$class);
```

Comment:
If this is only for `Highcharts` Component, there is no need for selecting the correct component, we can define it straight away.